scripts/generate-feature-docs.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+go install github.com/raviqqe/gherkin2markdown@latest
```

Comment:
It looks like this library is not generating the documentation in full. The only other options I found were for generating to other formats (i.e. html, word, excel, pdf) but simple markdown would be preferred.

Reply:
Resolved with a fork and adding support for rules.